### PR TITLE
Phase 1 - Add flask server

### PR DESCRIPTION
**Pull request by ninochang**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+__pycache__
+.env
```
Comment (allanlei) [inline .gitignore:2]: From my comment before

```suggestion
__pycache__
.*
```

File: app.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from flask import Flask, request
+from ckip_transformers.nlp import CkipWordSegmenter, CkipPosTagger, CkipNerChunker
+
+
+class Config:
+  CKIP_DEVICE = os.environ.get('CKIP_DEVICE') or -1 
+  CKIP_TRANSFORMER_MODEL = os.environ.get('CKIP_TRANSFORMER_MODEL')
+
+app = Flask(__name__)
+app.config.from_object(Config())
```
Comment (allanlei) [inline app.py:13]: No, not like this

File: app.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    model=app.config['CKIP_TRANSFORMER_MODEL'],
+)
+pos_driver = CkipPosTagger(
+  device=app.config['CKIP_DEVICE'], 
+  model=app.config['CKIP_TRANSFORMER_MODEL'],
+)
+ner_driver = CkipNerChunker(
+  device=app.config['CKIP_DEVICE'], 
+  model=app.config['CKIP_TRANSFORMER_MODEL'],
+)
```
Comment (allanlei) [inline app.py:26]: Compress this

File: app.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+@app.route('/tokenize', methods=['GET'], endpoint='tokenize')
+def tokenize():
+  tokens = ws_driver([request.args['text']])
```
Comment (allanlei) [inline app.py:30]: We discussed multipl value support

**Review by allanlei — CHANGES_REQUESTED** [review]

**Review by allanlei — CHANGES_REQUESTED** [review]
Where's the pip file for dependencies?

File: app.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+import os
+
+from flask import Flask, request
+from ckip_transformers.nlp import CkipWordSegmenter, CkipPosTagger, CkipNerChunker
+
+
+class Config:
+  CKIP_DEVICE = os.environ.get('CKIP_DEVICE') or -1 
+  CKIP_TRANSFORMER_MODEL = os.environ.get('CKIP_TRANSFORMER_MODEL')
```
Comment (allanlei) [inline app.py:10]: If it's none, which model would it be?

**Review by allanlei — CHANGES_REQUESTED** [review]
As discussed.  Use poetry

File: app.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+from flask import Flask, request
+from ckip_transformers.nlp import CkipWordSegmenter, CkipPosTagger, CkipNerChunker
+
+
+class Config:
+  CKIP_DEVICE = os.environ.get('CKIP_DEVICE') or -1 
+  CKIP_TRANSFORMER_MODEL = os.environ.get('CKIP_TRANSFORMER_MODEL') or 'bert-base'
+
+app = Flask(__name__)
+app.config.from_object(Config())
```
Comment (allanlei) [inline app.py:13]: Still no

**Review by allanlei — CHANGES_REQUESTED** [review]

File: app.py (hunk 6 above)
Comment (hiimdoublej-swag) [inline app.py:13]: I don't think we have to setup flask config if the values are just for ourselves to use.
```suggestion
CONFIG = {
  'CKIP_DEVICE' :os.environ.get('CKIP_DEVICE') or -1 
  'CKIP_TRANSFORMER_MODEL': os.environ.get('CKIP_TRANSFORMER_MODEL') or 'bert-base'
}
```

**Review by allanlei — APPROVED** [review]

**Pull request merged**